cmd/minikube/cmd/docker-env.go

```diff
@@ -287,14 +288,24 @@ docker-cli install instructions: https://minikube.sigs.k8s.io/docs/tutorials/doc
 		if len(co.Config.Nodes) > 1 {
 			exit.Message(reason.EnvMultiConflict, `The docker-env command is incompatible with multi-node clusters. Use the 'registry' add-on: https://minikube.sigs.k8s.io/docs/handbook/registry/`)
 		}
+		cr := co.Config.KubernetesConfig.ContainerRuntime
+		// nerdctld supports amd64 and arm64
+		if err := dockerEnvSupported(cr, driverName); err != nil {
+			exit.Message(reason.Usage, `The docker-env command is only compatible with the "docker" or "containerd" runtime on amd64/arm64 architectures, but this cluster is  configured to use the "{{.runtime}}" runtime on {{.arch}}.`,
+				out.V{"runtime": cr, "arch": runtime.GOARCH})
```

Comment:
```suggestion
			exit.Message(reason.Usage, err.Error())
```
I think we should use the message from the error so the user knows what they did wrong.